contrib/automation_tests/test_cases/capture_window.py

```diff
@@ -136,52 +136,37 @@ def _execute(self, track_index: int = 0, new_index: int = 0, expected_new_index:
                        "Expected track index {} after reordering, got {}".format(expected_new_index, index))
 
 
-class MatchTracks(CaptureWindowE2ETestCaseBase):
+class VerifyTracksExist(CaptureWindowE2ETestCaseBase):
     """
     Verify that the existing visible tracks match the expected tracks
     """
 
-    def _execute(self, expected_count: int = None, expected_names: List[str or Iterable[str]] = None,
-                 allow_additional_tracks=False):
+    def _execute(self, track_names: str or List[str or Iterable[str]] = None, allow_duplicates=False):
         """
         You need to pass either expected_track_count, or expected_name_list. If both are passed, they need to match
         w.r.t. expected number of tracks.
-        :param expected_count: # of tracks to be visible
-        :param expected_names: List of (partial) matching names of tracks to be visible.
+        :param expected_names: List of (partial) matching names of tracks to be visible, or a single track name.
             The name is a partial match, but the exact number of tracks is expected. Each entry in this list can either
             be a string or a list of strings. If an entry is a list of strings, the test expects at least on of the
             names to be matched (i.e. this is an "or" condition on multiple possible track names)
-        :param allow_additional_tracks: If True, encountering additional tracks beyond the given list / number is not
-            considered an error
```

Comment:
This is outdated, isn't it?

Reply:
And a wish: It would be great if you could mention that the track name matching uses `fnmatch` in its implementation.

Reply:
See above, and added the `fnmatch` hint

Reply:
Thanks.